Federated Tables: Quickstart guide and skeleton to continue (#15311)
CartoDB/cartodb: doc/developer-center/federated-tables-api/support/02-contribute.md

## Contribute

CARTO platform is an open-source ecosystem. You can read about the [fundamentals]({{site.fundamental_docs}}/components/) of CARTO architecture and its components.
We are more than happy to receive your contributions to the code and the documentation as well.

## Filling a ticket

If you want to open a new issue in our repository, please follow these instructions:

1. Descriptive title.
2. Write a good description, it always helps.
3. Specify the steps to reproduce the problem.
4. Try to add an example showing the problem.

## Contributing code

> Best part of open source, collaborate in Import API code!. We like hearing from you, so if you have any bug fixed, or a new feature ready to be merged, those are the steps you should follow:

Reviewer: ```suggestion
Best part of open source, collaborate in Federated Tables API code!. We like hearing from you, so if you have any bug fixed, or a new feature ready to be merged, those are the steps you should follow:
```

1. Fork the repository.
2. Create a new branch in your forked repository.
3. Commit your changes. Add new tests if it is necessary.
4. Open a pull request.
5. Any of the maintainers will take a look.
6. If everything works, it will merged and released \o/.

If you want more detailed information, this [GitHub guide](https://guides.github.com/activities/contributing-to-open-source/) is a must.

## Completing documentation

> Import API documentation is located in ```docs/```. That folder is the content that appears in the [Developer Center](http://carto.com/developers/import-api/). Just follow the instructions described in [contributing code](#contributing-code) and after accepting your pull request, we will make it appear online :).

Reviewer: ```suggestion
Federated Tables API documentation is located in ```docs/```. That folder is the content that appears in the [Developer Center](http://carto.com/developers/federated-tables-api/). Just follow the instructions described in [contributing code](#contributing-code) and after accepting your pull request, we will make it appear online :).
```

> Import API documentation is located in ```docs/```. That folder is the content that appears in the [Developer Center](http://carto.com/developers/import-api/). Just follow the instructions described in [contributing code](#contributing-code) and after accepting your pull request, we will make it appear online :).

Reviewer: ```suggestion
Federated Tables API documentation is located in ```docs/```. That folder is the content that appears in the [Developer Center](http://carto.com/developers/import-api/). Just follow the instructions described in [contributing code](#contributing-code) and after accepting your pull request, we will make it appear online :).
```
